### DISCUSSION TITLE
Update everything except specifiek images. how?

### DISCUSSION THREAD

**Discussion by 1mfaasj** [containrrr/watchtower#821] [Q&A] [answered]
I want all images to be updated (I think is standard behavior anyway, right?) EXCEPT specific images ... how do I get this done?

do I have to provide this environment?
WATCHTOWER_LABEL_ENABLE=FALSE

but what about the label?
com.centurylinklabs.watchtower.enable

**Comment by piksel** [comment, marked as answer]
Use the label `com.centurylinklabs.watchtower.enable` on the containers you want to skip (set to false), the environment variable is for the "global" behaviour of watchtower:
https://containrrr.dev/watchtower/arguments/#filter_by_enable_label
If that is used, it will skip all containers that does NOT have the label (set to true), which is the opposite of what you want.

Simply adding the label (and not specifying that environment variable (or it's command line counterpart)) is enough for it to skip them.

**Reply by 1mfaasj** [reply]
> com.centurylinklabs.watchtower.enable

thanks for the reply, so I only have to add this label to earch container I want to skip updating? Like this?

com.centurylinklabs.watchtower.false

or like this?
com.centurylinklabs.watchtower.enable
but this seems strange right? an enable label to skip the update process?

**Reply by 1mfaasj** [reply]
ah I understand ( I think), only this line should be enough, right?

      labels:
        - "com.centurylinklabs.watchtower.enable=false"

**Reply by piksel** [reply]
Yeah, I think that's the correct docker compose syntax.
